packages/node_modules/glimmer-syntax/lib/parser/handlebars-node-visitors.ts

```diff
@@ -107,7 +107,11 @@ export default {
       changeLines = leadingNewlineDifference(content.original, content.value);
     }
 
-    this.tokenizer.line = this.tokenizer.line + changeLines;
+    if (changeLines) {
```

Comment:
AFAIK, `leadingNewlineDifference` is not guaranteed to be > 0 (unless you know for certain that `content.rightStripped` guarantees that a newline was stripped?).  That is why I added a second conditional...